waffle/models.py

```diff
@@ -243,6 +243,19 @@ def _is_active_for_language(self, request):
 
     def is_active(self, request):
         if not self.pk:
+            if get_setting('CREATE_MISSING_FLAGS'):
+                try:
+                    with transaction.atomic():
```

Comment:
Why are you creating a transaction for all of these?

Reply:
In Postgres (not sure about other databases) if a cursor throws an IntegrityError then later interactions with the database will fail (even if you catch the exception) [1]. So I'm just trying to make sure that failure to create flags doesn't take down what would otherwise be a perfectly healthy request.

1. https://docs.djangoproject.com/en/2.1/topics/db/transactions/#controlling-transactions-explicitly

Although, upon further thought I could probably accomplish a similar thing by using `get_or_create` instead of just `create`.